packages/server/tests/previewnetAcceptance.env

```diff
@@ -0,0 +1,42 @@
+HEDERA_NETWORK=previewnet
+CHAIN_ID=0x129
+MIRROR_NODE_URL=https://previewnet.mirrornode.hedera.com
+LOCAL_NODE=false
+MIRROR_NODE_URL_WEB3=https://previewnet.mirrornode.hedera.com
+SERVER_PORT=7546
+RELAY_ENDPOINT='http://127.0.0.1:7546'
+# cypress metamask config
```

Comment:
Should set `E2E_RELAY_HOST` as the desired HashIO link
This will ensure the tests don't spin up their own instance of the relay